### DISCUSSION TITLE
How to format code block in markdown automatically

### DISCUSSION THREAD

**Discussion by kayx23** [facebook/docusaurus#8777] [Q&A]
Hello people! 

Anyone know any way to configure a formatter with docusaurus to automatically update/enforce code block indentation to two whitespaces in markdown files?

Question x-posted in discord but not receiving any response (yet). Not seeing this question asked anywhere. 

TIA!

**Comment by slorber** [comment]
Prettier has support for formatting markdown and MDX files, including code blocks

We use it ourselves on our docs

https://prettier.io/
